src/commands/compile_contract.rs

```diff
@@ -17,7 +17,7 @@ pub struct CompileContract {
     pub sierra_path: PathBuf,
 
     /// Path to where casm json file will be saved.
-    /// It will be serialized cairo_lang_starknet::casm_contract_class::CasmContractClass
+    /// It will be serialized `cairo_lang_starknet::casm_contract_class::CasmContractClass`
```

Comment:
```suggestion
    /// It will be serialized [`cairo_lang_starknet::casm_contract_class::CasmContractClass`]
```

?

Reply:
why tho

Reply:
so this will be presented as a clickable link in docs